checkstyle/import-control.xml

```diff
@@ -63,6 +63,9 @@
       <allow pkg="org.apache.kafka.common.config" />
       <allow pkg="org.apache.kafka.common.metrics" />
       <allow pkg="org.apache.kafka.common.security" />
+      <!-- for testing -->
+      <allow pkg="org.apache.kafka.common.requests" />
+      <allow pkg="org.apache.kafka.clients" />
```

Comment:
The first new `allow` will go away after the rebase. It would be good to figure out if we can remove the second new `allow`. `network` should not be accessing `clients` and we really want to enforce that for non-test code.